app/controllers/catalog_controller.rb

```diff
@@ -28,7 +28,7 @@ class CatalogController < ApplicationController
       qt: 'search',
       rows: 10,
       qf: 'title_tesim description_tesim creator_tesim keyword_tesim',
-      fq: '{!term f=has_model_ssim v=Work}'
+      fq: q: '{!term f=has_model_ssim v=collection OR work}'
```

Comment:
I think this is a typo. Pretty sure you don't want that stray `q:` there.

Reply:
@val99erie  suggested this `fq: '{!terms f=has_model_ssim v=Work,Collection}'` 
which works. :)